packages/backend/src/assets/ai.json

```diff
@@ -69,6 +69,23 @@
       ],
       "backend": "llama-cpp"
     },
+    {
+      "id": "chatbot-java",
```

Comment:
```suggestion
      "id": "chatbot-java-quarkus",
```

so it'll match suggested `exampleName-language-framework/platform`

Reply:
@rhatdan / @jeffmaury agree with renaming this recipe (I'd first change it in https://github.com/containers/ai-lab-recipes for consistency)?

Reply:
yes please change first into ai-lab-recipes for consistency